pkg/operator/controller/prometheus.go

```diff
@@ -177,7 +177,7 @@ func getAlertRules(runbookURLTemplate string) []promv1.Rule {
 		),
 		generateAlertRule(
 			"CDIDataVolumeUnusualRestartCount",
-			"kubevirt_cdi_import_dv_unusual_restartcount_total > 0 or kubevirt_cdi_upload_dv_unusual_restartcount_total > 0 or kubevirt_cdi_clone_dv_unusual_restartcount_total > 0",
+			"kubevirt_cdi_import_pods_high_restart > 0 or kubevirt_cdi_upload_pods_high_restart > 0 or kubevirt_cdi_clone_pods_high_restart > 0",
 			"5m",
 			map[string]string{
 				"summary":     "Cluster has DataVolumes (PVC population request) with an unusual restart count, meaning they are probably failing and need to be investigated",
```

Comment:
@akalenyu Is the alert message still correct? since its mentioning DataVolumes

Reply:
We can drop datavolumes since the introduction of populators so maybe
```
Some CDI population workloads have an unusual restart count, meaning they are probably failing and need to be investigated
```

Reply:
Done